### ISSUE TITLE
Fix the alignment of text in Login Screen

### ISSUE THREAD

**Issue by letsintegreat** [letsintegreat/Tagify#6]
**Important: This Issue is only for beginners to Open Source and Flutter. If you have some experience in either of these, go for another issues.**

![WhatsApp Image 2022-12-05 at 00 49 48](https://user-images.githubusercontent.com/37345795/205510884-b098b2ea-1969-42e9-8d5c-0af6c9b9dc89.jpg)

In `login_screen.dart`

The text "Login with Google" is aligned to the center of button. Instead make it align to the center of the remaining space after excluding the space occupied by the Google Icon.

Ask to be assigned in the comments below before starting to work on this issue. Stick to the contribution guidelines as mentioned in the README file.

**Comment by chandansgowda** [comment]
Hey @letsintegreat 
I think #8 , #7 and this issue can be merged into one issue since all of them has work to be done on the same widget.

**Comment by chandansgowda** [comment]
Oh okayy

**Comment by Tushar98644** [comment]
@letsintegreat can u assign me this?

**Comment by letsintegreat** [comment]
@Tushar98644 Assigned! As this is marked as a good first issue, if you need any help, don't hesitate to ping me on our server :)

**Comment by Tushar98644** [comment]
> @Tushar98644 Assigned! As this is marked as a good first issue, if you need any help, don't hesitate to ping me on our server :)

ya sure ....thanks
